packages/grid-pro/test/edit-column-renderer.test.js

```diff
@@ -247,7 +247,7 @@ describe('edit column renderer', () => {
 
       dblclick(cell._content);
       getCellEditor(cell).value = 'Bar';
-      grid.scrollToIndex(29);
+      grid.scrollToIndex(39);
```

Comment:
In Firefox, with `src` styles the item with `29` index was still in the viewport at the top of thegrid, and the check below failed as the `<input>` element was not removed. Updated the test to use more items.